jobs/otel-collector/templates/config.yml.erb

```
@@ -44,7 +44,7 @@ def check_for_unused_secrets!(secrets, used_secrets)
 end
 
 def check_for_new_and_old_properties!
-  both_provided = !p('config').empty? && (!p('metric_exporters').empty? || !p('trace_exporters').empty?)
+  both_provided = (!p('config').empty? || !p('prom_exporter_config').empty?) && (!p('metric_exporters').empty? || !p('trace_exporters').empty?)
```

We should bind the configuration to the selection of the other exporters. Can you please adapt it?

```suggestion
  both_provided = (!p('config').empty? && (!p('prom_exporter_config').empty?) || !p('metric_exporters').empty? || !p('trace_exporters').empty?)
```